src/View/base.html.twig

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+
+    <meta charset="UTF-8">
+    <title>
+        {% block title %}
+        Quizzy
+        {% endblock %}
+    </title>
+
+    {% block stylesheets %}
+    <link rel = "stylesheet" href = "mainStyle.css">
```

Comment:
/public/mainStyle.css doesn't exists
There's already a /public/assets/css directory where the simple-mvc framework seems to store its css files.

Reply:
updated